doc/release-process.md

```diff
@@ -61,62 +63,97 @@ The release script has the following dependencies:
 You can optionally install the `progressbar2` Python module with pip to have a
 progress bar displayed during the build process.
 
-## Release process
+## Versioning
 
-In the commands below, <RELEASE> and <RELEASE_PREV> are prefixed with a v, ie.
-v1.1.0 (not 1.1.0).
+Zcashd uses semantic version numbers as described by https://semver.org/.
 
-### Create the release branch
+## Release candidate & release process
 
-Run the release script, which will verify you are on the latest clean
-checkout of master, create a branch, then commit standard automated
+Identify the commit from which the release stabilization branch will be made.
+Release stabilization branches are used so that development can proceed
+unblocked on the `master` branch during the release candidate testing and
+bug-fixing process. By convention, release stabilization branches are named
+`version-X.Y.0` where `X` and `Y` are the major and minor versions for the
+release.
+
+In the commands below, <RELEASE> and <RELEASE_PREV> must identify `git` tags
+prefixed with the character `v`, i.e. `v1.0.9` (not `1.0.9`). <COMMIT_ID> is a
+`git` hash identifying the commit on which a release stabilization or release
+branch will be based. It is recommended to use the entire hash value to
+identify the commit, although a prefix of at least 10 characters is also
+permitted.
+
+### Create the release stabilization branch
+
+Having identified the commit from which the release will be made, the release
+manager constructs the release stabilization branch as follows:
+
+    $ git checkout -b version-X.Y.0 <COMMIT_ID>
+    $ git push 'git@github.com:zcash/zcash' $(git rev-parse --abrev-ref HEAD)
+
+### Create the release candidate branch
+
+Run the release script to create the first release candidate. This will create
+a branch based upon the specified commit ID, then commit standard automated
 changes to that branch locally:
 
-    $ ./zcutil/make-release.py <RELEASE> <RELEASE_PREV> <RELEASE_FROM> <APPROX_RELEASE_HEIGHT>
+    $ ./zcutil/make-release.py <COMMIT_ID> <RELEASE> <RELEASE_PREV> <RELEASE_FROM> <APPROX_RELEASE_HEIGHT>
 
 Examples:
 
-    $ ./zcutil/make-release.py v1.1.0 v1.0.0 v1.0.0 120000
-    $ ./zcutil/make-release.py v1.1.0 v1.1.0-rc1 v1.0.0 222900
+    $ ./zcutil/make-release.py a410f124b2 v1.0.9 v1.0.8-1 v1.0.8-1 120000
+    $ ./zcutil/make-release.py 3ee1d04a8d v1.0.13 v1.0.13-rc1 v1.0.12 222900
```

Comment:
I think there's a rebase bug here; these verisons were altered in a previous PR to match what releases will now use.
```suggestion
    $ ./zcutil/make-release.py a410f124b2 v1.1.0 v1.0.0 v1.0.0 120000
    $ ./zcutil/make-release.py 3ee1d04a8d v1.1.0 v1.1.0-rc1 v1.0.0 222900
```